charter.md

```diff
@@ -46,6 +46,8 @@ The following organizations make up the Interop Team, in alphabetical order:
 
 ## Joining and Leaving the Team
 
-If your organization has a track record of contributions to the web-platform-tests project, you may request to join the Interop Team by contacting interop@web-platform-tests.org. The decision will be made by the current team based on consensus.
+The Interop Team is made up of organization with a track record of contributions to the web-platform-tests project.
 
-Team members can be removed by their own request or the consensus of the entire team.
+Organization my apply to join the team by contacting interop@web-platform-tests.org. New members will be added with a consensus decision of existing members.
+
+Existing members may leave the team at any time without further obligation. If a member is no longer active in the team, they can be removed by consensus amongst the other team members.
```

Comment:
So as written, this doesn't handle the case where e.g. someone joins and then intentionally blocks all work, or they meet some threshold of being "active" but are not positively contributing to the goals of the project.

In practice I don't think that would happen, and if it did the other members would obviously find some workaround. But I'm still not sure it's worth adding in a conditional on the situations in which it's allowable to remove a member. If you really want a conditional it should at least be less restricitve than "no longer active".

Reply:
This is a more antagonistic case than I'd considered. We could say that troublemakers can be ejected with the unanimous agreement of every other member, perhaps.

But maybe it's better to leave it to the wpt core team to sort out a situation like this? I'll make the edit to drop this bit.